src/unix/linux-syscalls.c

```diff
@@ -358,7 +358,7 @@ int uv__statx(int dirfd,
               int flags,
               unsigned int mask,
               struct uv__statx* statxbuf) {
-#if defined(__NR_statx)
+#if defined(__NR_statx) && defined(__ANDROID__)==0
```

Comment:
Can you change the Android bits to: `!defined(__ANDROID__)`.

Reply:
fixup